common/testing/junit5/pom.xml

```diff
@@ -40,5 +40,10 @@
             <artifactId>junit-jupiter-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
This does not make sense here - there is no `logging.properties`, so the logging is not required.

Reply:
As the issue describes, this should only be added to modules that use our appender. Please make sure that is the case for all added dependencies.